scripts/list_updatable_packages

```diff
@@ -56,8 +56,9 @@ def main() -> None:
 
     if 'GITHUB_ACTION' in os.environ:
         directories = [entry['directory'] for entry in matrix]
-        print(f'::set-output name=directories::{json.dumps(directories)}')
-        print(f'::set-output name=matrix::{json.dumps(matrix)}')
+        with open(os.environ['GITHUB_OUTPUT'], 'w') as github_output:
```

Comment:
Should it be `w` or `a` as mode? The example is `echo .. >> $GITHUB_OUTPUT` so that suggests `a` to me, but they failed to provide a programmatic example in their blog and documentation.

For this particular script it may not matter, but I'd like to know what is the right thing to do.